ui/src/DialogResponses.ts

```diff
@@ -17,4 +17,5 @@ export namespace DialogResponses {
     export const upload: MessageItem = { title: localize('upload', "Upload") };
     export const alwaysUpload: MessageItem = { title: localize('alwaysUpload', "Always upload") };
     export const dontUpload: MessageItem = { title: localize('dontUpload', "Don't upload"), isCloseAffordance: true };
+    export const reportIssue: MessageItem = { title: localize('reportIssue', "Report issue") };

```

Comment:
I wonder if "Report an Issue" or "Report as Issue" would be better. This dialog will show for many things which are not bugs, and "Report Issue" to me implies we're saying it's an issue and you should report it.

Reply:
I like "Report an issue"